src/lib/github/fetchGitHubUser.ts

```diff
@@ -0,0 +1,38 @@
+import { getAccessToken } from '@/lib/auth/getAccessToken';
+import { getUserId } from '@/lib/auth/getUserId';
+import type { components } from '@octokit/openapi-types';
+
+// GitHub username is the `login` property
+export type GitHubUser = components['schemas']['simple-user'];
+
+export async function fetchGitHubUser(): Promise<GitHubUser> {
+  const emptyResponse = Promise.resolve({} as GitHubUser);
```

Comment:
I had the same comment as in `src/lib/github/fetchGitHubRepos.ts` - maybe this is something required by a library? I wasn't sure.